app/models/message/Message.php

```diff
@@ -123,11 +128,7 @@ public function set($stanza, $parent = false)
                 $this->body = trim(html_entity_decode($this->body));
             }
 
-            if($stanza->markable) {
-                $this->markable = true;
-            } else {
-                $this->markable = false;
-            }
+            $this->markable = $stanza->markable;
```

Comment:
Here it check if the XML tag is there, $stanza->markable is not a boolean

Reply:
What is the type of `$stanza`? I see no method `attributes` for [DOMElement](https://secure.php.net/manual/en/class.domelement.php).